RecoTracker/IterativeTracking/python/InitialStep_cff.py

```diff
@@ -71,10 +101,10 @@
     propagator = initialStepSeeds.propagator,
 
 )
-initialStepHitQuadruplets.SeedComparitorPSet = initialStepSeeds.SeedComparitorPSet
+_initialStepHitQuadrupletsMerging.SeedComparitorPSet = initialStepSeeds.SeedComparitorPSet
 
+trackingPhase1PU70.toReplaceWith(initialStepHitQuadruplets, _initialStepHitQuadrupletsMerging) 
```

Comment:
@slava77 
The "change" is to preserve the current behaviour for `trackingPhase1PU70`. Before, `initialStepHitQuadruplets` label pointed to `PixelQuadrupletMergerEDProducer`, while Erica changes it point to `PixelQuadrupletEDProducer` instead. So this line changes the label to point back to `PixelQuadrupletMergerEDProducer` for `trackingPhase1PU70`.


> Given limited scope of this era, it's probably not that important either way.

Indeed, I'll disable the `trackingPhase1PU70` workflows in a PR to be submitted soon (leaving the actual cleanup later to avoid conflicting developments for pre2).